pytorch_lightning/strategies/ddp.py

```diff
@@ -62,8 +62,9 @@
 
 if _FAIRSCALE_AVAILABLE:
     from fairscale.optim import OSS
-if _TORCH_GREATER_EQUAL_1_10:
-    from torch.distributed.algorithms.model_averaging.averagers import ModelAverager
+if _TORCH_GREATER_EQUAL_1_10 and torch.distributed.is_available():
+    if torch.distributed.is_available():
+        from torch.distributed.algorithms.model_averaging.averagers import ModelAverager
```

Comment:
```suggestion
if _TORCH_GREATER_EQUAL_1_10 and torch.distributed.is_available():
    from torch.distributed.algorithms.model_averaging.averagers import ModelAverager
```
??

Reply:
Good catch. Something went wrong with the previous suggestion. I didn't notice. Thanks!